app/graphql/mutations/events/create_event.rb

```diff
@@ -0,0 +1,19 @@
+module Mutations
+  module Events
+    class CreateEvent < Mutations::BaseMutation
+      argument :user_id, ID, required: true
+      argument :name, String, required: true
+      argument :date, String, required: true
+      argument :color, String, required: true
```

Comment:
Color isn't required for new event or era (it'll be optional)

Reply:
Updated!